src/server/chunk_server/chunk_server_file_service_impl.cc

```diff
@@ -23,17 +28,103 @@ namespace service {
 grpc::Status ChunkServerFileServiceImpl::InitFileChunk(
     ServerContext* context, const InitFileChunkRequest* request,
     InitFileChunkReply* reply) {
-  // TODO(someone): implement the GFS chunk server logic here
+  LOG(INFO) << "Received InitFileChunkRequest:" << (*request).DebugString();
   *reply->mutable_request() = *request;
```

Comment:
So far I haven't seen client taking advantage of know the request from the reply message, I presume the original design rationale for this is for async processing.